context/journal/2026-05-21.md

```
@@ -0,0 +1,5 @@
+---
+timestamp: 2026-05-21T04:53:43Z
+entry_type: completion
+---
+Investigated browser screenshot rejection. Root cause was PNG screenshot files transcoded to JPEG bytes but labeled with the original PNG MIME in OpenAI/OpenRouter data URIs; fixed adapters to emit `data:image/jpeg` and added regression tests.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)